qiskit/providers/ibmq/api_v2/websocket.py

```diff
@@ -30,6 +31,9 @@
 
 logger = logging.getLogger(__name__)
 
+# Patch asyncio to allow nested use of `loop.run_until_complete()`.
```

Comment:
Before merging, can we add a comment at the point we use `run_until_complete` saying that without this dependency, it would not work under a Jupyter environment?